folium/plugins/semicircle.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import Figure, JavascriptLink
+
+from folium.map import Marker
+from folium.utilities import _validate_location
+
+from jinja2 import Template
+
+
+class SemiCircle(Marker):
+    """
+    Creates a Semicircle plugin to append into a map with
+    Map.add_plugin.
+    Use (direction and arc) or (startAngle and stopAngle)
+    Parameters
+    ----------
+    location: tuple of length 2, default None
+        The latitude and longitude of the marker.
+        If None, then the middle of the map is used.
+    radius: int, default 0
+        Radius of semicircle
+    direction: int, default 0
+        Heading of direction angle value between 0 and 360 degrees
+    arc: int, default 0
+        Heading of arc angle value between 0 and 360 degrees.
+    startAngle: int, default 0
+        Heading of the start angle value between 0 and 360 degrees
+    stopAngle: int, default 0
+        Heading of the stop angle value between 0 and 360 degrees.
+    """
```

Comment:
Add a line that `**kwargs` are passed to `Marker`, so a user can provide a tooltip and popup.